crates/configuration/Cargo.toml

```
@@ -18,6 +18,7 @@ sqlx = { workspace = true, features = ["json", "postgres", "runtime-tokio-rustls
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 tracing = { workspace = true }
+prometheus.workspace = true
```

Can you follow the format above, and sort the entries?

```suggestion
prometheus = { workspace = true }
```

---

Sure. Didn't notice what the actual change was because I had `cargo add prometheus` do it.